Translation corrections (#428)
07th-mod/higurashi-console-arcs: Update/ura_007.txt

			NULL, " Things have been going poorly with your friends recently.", Line_WaitForInput);
	ModPlayVoiceLS(4, 36, "ps3/s14/36/vt4bb_bnatsu1053", 256, TRUE);
	OutputLine(NULL, "なれない勉強続きで、そろそろ疲れが限界点…」",

> 			NULL, " And with all this studying of stuff you don't know, your fatigue's about to hit its limit...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " And with all this studying you're still not used to, your fatigue's about to hit its limit...\"", Line_Normal);
```
